[Service Bus] ATOM API - Rename the listing APIs from "get" to "list" (#10223)
Azure/azure-sdk-for-js: sdk/servicebus/service-bus/CHANGELOG.md

### Breaking Changes

> - Added Async iterable iterators with pagination support for all the listing methods like `listQueues()`, `listTopics()`,`listQueuesRuntimeInfo()`, etc.
>   The equivalent APIs that used to return arrays like `getQueues`, `getTopics()`, `getQueuesRuntimeInfo()` are not exported anymore.

Reviewer: ```suggestion
- Added Async iterable iterators with pagination support for all the listing methods like `getQueues()`, `getTopics()`,`getQueuesRuntimeInfo()`, etc. and renamed them to use the `list` verb.
```

Author: Nice.

### Breaking Changes

> - Added Async iterable iterators with pagination support for all the listing methods like `getQueues()`, `getTopics()`,`getQueuesRuntimeInfo()`, etc. and renamed them to use the `list` verb.

Author: ```suggestion
- Added Async iterable iterators with pagination support for all the listing methods like `getQueues()`, `getTopics()`, `getQueuesRuntimeInfo()`, etc. and renamed them to use the `list` verb.
```
